packages/playground/app/_layouts/default.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+
+<html lang="{{ site.lang | default: "en-US" }}">
+{% include head.html %}
+<body  role="application">
```

Comment:
you have body tag here but as well in all .sample.html files. Do we have two body tags in the result HTML?

consider removing the body tag from the .sample.html files